### ISSUE TITLE
0.9.1 - DateTime is better but some issue is still there

### ISSUE THREAD

**Issue by murbanowicz** [quasarframework/quasar#229]
DATETIME - it is better but it's not fully fixed still and I can't find root cause.
When it is rendered with property which is not defined before render, it is now responding (you can choose date and it is visible in calendar popup) and it is changing model value, but IT IS NOT CHANGING QDateTime value which keeps the same value. When I will go back from edition of element and then go to edition again, it is working properly (setting model and setting value).

There is good progress but something is still wrong. I do not blame you because almost every datetime component around has some issues. It looks like we are almost there but some small issue still exist.

**Comment by rstoenescu** [comment]
Can you rephrase that pls? I don't know what you mean by "it is changing model value but not changing QDateTime". Also what is edition of element?

**Comment by murbanowicz** [comment]
Sorry!

I have list of elements. Whole div with edit form is hidden with `v-if="editedElement"`
Quasar DateTime `v-model="editedElement.someDate"`. Going to edition means that I am setting some element as `editedElement` (so then Quasar components are getting some value).

By setting model and not changing value, I mean that model is being changed - `editedElement.someDate` but `value` of the `QDateTime` is not being changed which is causing that it is not displayed properly. 
Hope it helps?

**Comment by rstoenescu** [comment]
I'm sorry but I still don't get it. Can you please make a .vue file to isolate the issue with bare minimum working code and no deps to other parts of the code? Sometimes code speaks better than words :)

**Comment by rstoenescu** [comment]
@murbanowicz Any progress?

**Comment by rstoenescu** [comment]
Please reopen with .vue file. Thanks.
